.env.example

```diff
@@ -18,3 +18,7 @@ LOG_LEVEL=debug
 # Nodemailer config
 NOTIFICATIONS_EMAIL_USER="no-reply@senecacollege.ca"
 MAILHOG_SMTP_PORT=1025
+SAML_IDP_METADATA=http://localhost:8081/simplesaml/saml2/idp/metadata.php
```

Comment:
Thinking about this a bit more.  It might be better to have all of the exports from the `saml.server.ts` module be `async`.  This would allow us to pull the IdP metadata on startup and use that (i.e., first request would get the metadata and config the idp, subsequent runs would be faster since we already have it).

Reply:
I am trying the following but it is claiming that neither `http://localhost:8081/simplesaml/saml2/idp/metadata.php` or `http://host.docker.internal:8081/simplesaml/saml2/idp/metadata.php` exist
```
//Take the metadata stood up by the IDP and use it as the metadata for our IDP object
const idp = samlify.IdentityProvider({
  metadata: readFileSync(`${process.env.SAML_IDP_METADATA}`),
});
```

What am I doing wrong?

Reply:
Wait, I think I misunderstood, you would like to have the idp metadata stored as a file local to starchart, is that correct?

Reply:
In the last commit I had it read from the idp config locally, when I try to load idp once in saml server at the top level it gives me the error that await can not be used at the top level.